ads/google/a4a/utils.js

```diff
@@ -148,6 +167,9 @@ export function googleAdUrl(
         {name: 'brdim', value: additionalDimensions(win, viewportSize)},
         {name: 'isw', value: viewportSize.width},
         {name: 'ish', value: viewportSize.height},
+        {name: 'ea', value: '0'},
+        {name: 'pfx', value: 'fc' in containerTypeSet
```

Comment:
'in' makes me nervous and I suggest referring to ValidAdContainerTypes:

value: containerTypeSet[ValidAdContainerTypes['AMP-FX-FLYING-CARPET']] || containerTypeSet[ValidAdContainerTypes['AMP-STICKY-AD']]